en/editor/publish/gpg-input-sdk.md

```diff
@@ -0,0 +1,150 @@
+# Integrating Input SDK
+
+The Input SDK provides a unified interface that allows players to easily find the mouse and keyboard bindings for your game on Google Play Games for PC, thereby enhancing their gaming experience.
+
+## Prerequisites
+
+If you are using Input SDK on a new Android build, simply check the Input SDK option in the build panel.
+
+If the Android project was built previously, you will need to manually add the following code to the `native/engine/android/app/build.gradle` file.
+
+```gradle
+dependencies {
+  implementation 'com.google.android.libraries.play.games:inputmapping:1.0.0-beta'
+  ...
+}
+```
+
+## Create a the class named `MyInputMapProvider`
+
+Create a new file `MyInputMapProvider.java` which contains the following code:
```

Comment:
Please change to "Create a new file 'MyInputMapProvider.java'. For this example, this is what should be contained in this code:"